# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/crashtests/crashtests.list
# Commit: 0475e2855e5c
# Full Hash: 0475e2855e5c4b03dfce9da151fd4ac2bb1367c3
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1534394
# File Overlap Count: 1
# Description:
#   Bug 1534394 - Make WSRunObject::InsertText() set aPointAfterInsertedString by itself when HTMLEditor::InsertTextWithTransaction() returns error r=m_kato
#   
#   Oddly, `WSRunObject::InsertText()` returns `NS_OK` even when
#   `HTMLEditor::InsertTextWithTransaction()` returns error.  However,
#   it fails if insertion point is not editable like `<noscript>` element.
# ==============================================================================

diff -r 4cd6d838796a -r 0475e2855e5c editor/libeditor/crashtests/crashtests.list
--- a/editor/libeditor/crashtests/crashtests.list	Thu May 23 06:35:52 2019 +0000
+++ b/editor/libeditor/crashtests/crashtests.list	Thu May 23 06:06:18 2019 +0000
@@ -108,4 +108,5 @@
 asserts(0-2) load 1464251.html # assertion is that mutation event listener modifies content
 pref(layout.accessiblecaret.enabled,true) load 1470926.html
 load 1525481.html
+load 1534394.html
 load 1547898.html
